src/OrchardCore/OrchardCore.DisplayManagement/Entities/SectionDisplayDriver.cs

```diff
@@ -12,6 +12,7 @@ namespace OrchardCore.DisplayManagement.Entities
     /// shape instance for a specific section of the object. A section represents a property of an entity instance
     /// where the name of the property is the type of the section.
     /// </summary>
+    /// <typeparam name="TModel">The type of model this driver handles.</typeparam>
```

Comment:
![Uploading image.png…]()

Reply:
What was this one @Piedone ? I can't see the image

Reply:
Oh well, it was a joke with Henry Ford and the Model T :(.

Reply:
Disappointed now :(